### PR TITLE
solved warning

### PR DESCRIPTION
**Pull request by satyammangroliya**

### CODE REVIEW

**Comment by xJuvi** [comment]
Hi,
i tried to fix this warning with PR #56 (reported by issue #55)

Your solution double checks the client_id. Directly after your changes the `initIlias` method was called where the same check is placed.

https://github.com/Minervis-GmbH/BigBlueButton-Ilias-Plugin/blob/83bd58fb289f52b17da3272e5aa0489347dba7b9/guest.php#L48-L54

We should remove one of the checks. But i think this repo isn't maintained anymore if you take a look into my open PRs.

Kind regards

File: guest.php [hunk 1]
```diff
@@ -463,8 +463,12 @@ private function httpExit(int $code = 404)
 
     private function __construct()
     {
-        $this->client = filter_var($_GET['client_id'], FILTER_SANITIZE_STRING);
-        $this->refId = filter_var($_GET['ref_id'], FILTER_SANITIZE_NUMBER_INT);
+        $this->client = isset($_GET['client_id']) ? filter_var($_GET['client_id'], FILTER_SANITIZE_STRING) : null;
+        $this->refId = isset($_GET['ref_id']) ? filter_var($_GET['ref_id'], FILTER_SANITIZE_NUMBER_INT) : 0;
+        if (is_null($this->client)) {
```
Comment (xJuvi) [inline guest.php:468]: Is already exist in `initIlias` :

https://github.com/Minervis-GmbH/BigBlueButton-Ilias-Plugin/blob/83bd58fb289f52b17da3272e5aa0489347dba7b9/guest.php#L50-L54

**Review by xJuvi — CHANGES_REQUESTED** [review]

**Comment by jeph864** [comment]
@xJuvi ,
I just merged your PR #56 Thank you

**Comment by jeph864** [comment]
@satyammangroliya ,
thanks for bringing this up. I have merged @xJuvi 's code.

**Pull request closed without merging**